.travis.yml

```diff
@@ -51,6 +51,27 @@ install:
       source run_conda_forge_build_setup
 
 script:
-  - conda build ./recipe
+  #################################################
+  #              BEGIN CUSTOM CHANGE              #
+  #                                               #
+  # This change needs to stay intact on rerenders #
+  #################################################
+
+  # The build contains hardcoded references to the work directory in the
+  # binary files. To fix this, we make the work directory exactly as long as
+  # the placeholder prefix (255 characters.
+
+  # The prefix ends with '/work/emacs-25.1' (16 characters). We put it in the
+  #  miniconda3 directory, '/Users/travis/miniconda3/' (25 characters), so our
+  # placeholder directory name needs 214 characters.
+
+  - CONDA_BLD_PATH=/Users/travis/miniconda3/placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac
```

Comment:
Can set this as part of [`conda-forge.yml`'s `matrix`]( https://github.com/conda-forge/tk-feedstock/blob/1ddb62047b5dc0bd31928c38f66f09bca5534cfd/conda-forge.yml#L1-L4 ) to make life a little easier.

Reply:
But I need to do a binary replacement of it in the package. So unless there's some way to extract it (?), I need to hard code it to something that I can later replace.

Reply:
Is the conda-forge.yml matrix flag documented somewhere? This environment variable should be set only for Travis.